bom/application/pom.xml

```diff
@@ -189,6 +191,7 @@
         <apicurio-registry.version>2.2.4.Final</apicurio-registry.version>
         <apicurio-common-rest-client.version>0.1.9.Final</apicurio-common-rest-client.version> <!-- must be the version Apicurio Registry uses -->
         <jacoco.version>0.8.8</jacoco.version>
+        <javassist.version>3.27.0-GA</javassist.version>
```

Comment:
Where is this property being used?

Reply:
Ups, thanks for catching this. Apparently the change in the BOM is missing.